identity-account/src/storage/stronghold.rs

```diff
@@ -182,15 +190,19 @@ impl Storage for Stronghold {
     let store: Store<'_> = self.store(&fmt_did(did));
 
     // Read the state from the stronghold snapshot
-    let data: Vec<u8> = store.get(location_state()).await?;
-
-    // No state data found
-    if data.is_empty() {
-      return Ok(None);
+    let data: Option<Vec<u8>> = store.get(location_state()).await?;
+
+    match data {
+      None => return Ok(None),
+      Some(data) => {
+        // No state data found
+        if data.is_empty() {
+          return Ok(None);
+        }
```

Comment:
I think since `Store::get` returns `None` if the key wasn't found, this doesn't need to be handled anymore, the way it was previously. In other words, we want `IdentityState::from_json_slice` to error if `data.is_empty()` because that indicates a storage error.

Reply:
I removed the `.is_empty()` check.